website/docs/cli/commands/test.mdx

```diff
@@ -43,3 +45,13 @@ The Terraform `test` command creates _real_ infrastructure. Once Terraform fully
 You should monitor the output of the test command closely to ensure Terraform removes the infrastructure it created or perform manual cleanup if not. We recommend creating dedicated testing accounts within the target providers that you can routinely and safely purge to ensure any accidental and costly resources aren't left behind.
 
 Terraform also provides diagnostics explaining why it could now automatically clean up. You should resolve these diagnostics to ensure that future clean-up operations are successful.
+
+## Terraform Cloud Execution
+
+You can execute your tests remotely on Terraform Cloud using the `-cloud-run` option.
+
+The argument accepts a private registry module source as specified in the [Terraform Registry Module Sources](/terraform/language/modules/sources#terraform-registry) documentation. This will associate the test run with the given private module within the Terraform Cloud UI.
+
+You must provide a module from a **private** registry and not the public Terraform registry.
```

Comment:
I like the bold, but we use italics for emphasis, so I think we should change to:
```suggestion
You must provide a module from a _private_ registry, not the public Terraform registry.
```